core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -28,13 +28,14 @@ private[spark] trait ExecutorAllocationClient {
    * This can result in canceling pending requests or filing additional requests.
    * @return whether the request is acknowledged by the cluster manager.
    */
-  private[spark] def requestTotalExecutors(numExecutors: Int): Boolean
+  private[spark] def requestTotalExecutors(numExecutors: Int,
```

Comment:
minor: since there's only one implementation at the moment, could we avoid adding a new method and instead just change the existing one?